packages/front-end/components/Admin/CreateOrganization.tsx

```diff
@@ -32,27 +31,11 @@ const CreateOrganization: FC<{
     <Modal
       submit={handleSubmit}
       open={true}
-      header={
-        isAdmin ? (
-          "Create New Organization"
-        ) : (
-          <img
-            alt="GrowthBook"
-            src="/logo/growthbook-logo.png"
-            style={{ height: 40 }}
-          />
-        )
-      }
+      header={"Create New Organization"}
```

Comment:
ah nice cleanup.  I had thought that this was used on the main login page, but now I see that is done in the other CreateOrganization.